### ISSUE TITLE
[BUG] Secret recovery phrase cannot select words with screen reader

### ISSUE THREAD

**Issue by brianacnguyen** [MetaMask/metamask-mobile#10089]
- Screen reader user could not complete onboarding. The SRP words here are not selectable so this was the end of their journey
    - Tab order will be relevant
    - How to repeat the bug
        - Turn on voice over
        - See how to test AT switch device for tab
<img width="970" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNWlJQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--14cc0fcdc25f63ab3e6be9245a97961725060500/image.png" alt="image.png" />
